utils/tests/test_fake_data.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+""" Tests utils/fake_data.py """
+import pytest
+
+from utils.fake_data import MilMoveProvider
+from utils.fake_data import MilMoveData
+from faker import Faker
+from faker.generator import Generator
+from datetime import datetime
+from utils.constants import DataType
+
+
+class TestMilMoveProvider:
+    """ Tests the MilMoveProvide class and its methods. """
+
+    @classmethod
+    def setup_class(cls):
+        """ Initialize the MilMoveProvide that will be tested. """
+        cls.fake = Faker()
+        cls.fake.add_provider(MilMoveProvider)
+        cls.provider = MilMoveProvider(Generator())
+
+    def test_init(self):
+        """
+        Tests that the __init__ from the setup_class method worked correctly.
+        """
+        assert self.provider.current_name == {"first_name": "", "last_name": ""}
+        assert self.provider.first_name_used is True
+        assert self.provider.last_name_used is True
+
+    def test_safe_phone_number(self):
+        """
+        Tests the returned phone number is the correct length and structure
+        """
+        phone_number = self.fake.safe_phone_number()
+        assert type(phone_number) is str
+        assert len(phone_number) == 12
+        assert phone_number.index("555") == 4
```

Comment:
This last assertion is a little goofy, haha. Are you comfortable with regex? This would be a good place to test that what we're getting back matches the pattern we expect to see, namely `[2-9]\d{2}-555-\d{4}`. Don't feel obligated to do pattern matching if it's new to you, but if you're curious about regex, this is a really good resource: https://regexone.com/